component/src/main/ballerina/net.ftp/ftp_server_connection.bal

```diff
@@ -0,0 +1,12 @@
+package ballerina.net.ftp;
```

Comment:
Lincence header?

Reply:
I have added to the new file. This local DirectoryListener module will move to the main ballerina repo.